wp-includes/widgets/class-wp-widget-image.php

```diff
@@ -33,6 +33,10 @@ public function __construct() {
 			'edit_media' => __( 'Edit Image' ),
 			'change_media' => __( 'Change Image' ),
 			'select_media' => __( 'Select Image' ),
+			'error' => sprintf(
+				__( 'We can&#8217;t find that image. Check your <a href="%s">media library</a> and make sure it wasn&#8217;t deleted.' ),
```

Comment:
Added in 43e5c93.
I decided to go with a new selector that is a little broader than I'd like it to be, in order to not give translators the chance to break functionality by accidentally omitting the `.select-media` class.